rest/utilities_testing.go

```diff
@@ -801,6 +805,17 @@ func (rt *RestTester) ReplacePerBucketCredentials(config base.PerBucketCredentia
 	rt.ServerContext().bootstrapContext.connection = couchbaseCluster
 }
 
+func (rt *RestTester) Context() context.Context {
```

Comment:
This is going to initialize a new context every time it's called - is that intentional?